graphql_compiler/cost_estimation/statistics.py

```diff
@@ -86,34 +114,52 @@ def get_class_count(self, class_name):
                         GraphQL schema.
 
         Returns:
-            - int, the count of vertex or edge instances with, or inheriting the given class name
+            - int, count of vertex or edge instances having, or inheriting, the given class name.
 
         Raises:
-            AssertionError, if statistic for the given vertex/edge class does not exist.
+            AssertionError, if the count statistic for the given vertex/edge class does not exist.
         """
         if class_name not in self._class_counts:
             raise AssertionError(u'Class count statistic is required, but entry not found for: '
                                  u'{}'.format(class_name))
         return self._class_counts[class_name]
 
-    def get_edge_count_between_vertex_pair(
+    def get_vertex_edge_vertex_count(
         self, vertex_source_class_name, edge_class_name, vertex_target_class_name
     ):
-        """Return the count of edges connecting two vertices.
+        """Return the count of edges of the given class connecting vertex_source to vertex_target.
 
-        This statistic is optional, as the estimator can roughly predict this cost using
+        This statistic is optional, as the estimator can roughly predict this statistic using
         get_class_count(). In some cases of traversal between two vertices using an edge connecting
-        the vertices' superclasses, the estimates generated using get_class_count() may be off by
-        several orders of magnitude. In such cases, this statistic should be provided.
+        the vertices' superclasses, the predicitons made using get_class_count() may be off
```

Comment:
Why change the wording here?

Reply:
My goal was to make it clearer that it's `your` job to ask for an estimate, this api function won't do it for you. I'll work on making this clearer.